src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -181,7 +183,7 @@ void CoreWorkerDirectTaskSubmitter::ReturnWorker(const rpc::WorkerAddress addr,
   }
 
   auto status = lease_entry.lease_client->ReturnWorker(
-      addr.port, addr.worker_id, was_error, worker_exiting);
+      addr.port, addr.worker_id, was_error, error_detail, worker_exiting);
```

Comment:
This file change is to add more detail when a worker is returned with was_error = true. I found it currently doesn't give any detail